content/en/docs/installation/system-requirements.md

```diff
@@ -0,0 +1,83 @@
+
+Note that Armory does not produce marketplace specific images that can be used by different certified Kubernetes offerings. 
+
+## Browsers
+
+The UI for Armory Enterprise works best on Firefox or Chromium-based browsers.
+
+## External storage
+
+Armory Enterprise requires external storage for storing metadata and history.
+
+### Bucket storage
+
+You need an S3-compatible object store, such as an S3 bucket or Minio, for persisting your application settings and pipelines. The account you use to install and run Armory Enterprise needs read/write access to the buckets.
+
+### RDBMS (SQL)
+
+Depending on the service, Armory Enterprise also uses either Redis, MySQL, or Postgres as a backing store. The following table lists the supported database and the  service:
+
+| Database | DB version             | Armory                 | Spinnaker services                                  | Note                                                                                                                       |
+| -------- | ---------------------- | ---------------------- | --------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| Redis    | All supported versions | All supported versions | All Spinnaker services that require a backing store | The DB versions refer to external Redis instances. Supported only for services that still require Redis for a feature, such as Gate sessions. Redis is not supported as a core persistent storage engine. Although Spinnaker deploys internal Redis instances, do not use these instances for production deployments. Armory recommends only using them for testing and proof-of-concept deployments. |
+| MySQL    | MySQL 5.7 (or Aurora)  | All supported versions | Clouddriver, Front50, Orca                          |                                                                                                                            |
+| PostgreSQL    | PostgreSQL 10.0 or later  | 2.24.0 or later | Clouddriver                          |                                                                                                                            |
+
+Armory recommends using MySQL or PostgreSQL as the backing store when possible for production instances of Spinnaker. For other services, use an external Redis instance for production instances of Spinnaker.
+
+## Hardware requirements
+
+Armory recommends a minimum of 3 nodes that match the following profile:
+
+* **CPUS**: 8
+* **Memory (GiB)**: 32
+
+## kubectl
+
+To install and manage Armory Enterprise, any actively maintained version of `kubectl` is supported unless you want to use the [Armory Kustomize Repo](https://github.com/armory/spinnaker-kustomize-patches). The Kustomize Repo is a collection of Kustomize files that help you configure Armory Enterprise. The repo supports the following versions of `kubectl`: 1.16 to 1.19.
```

Comment:
This may need rewording as the Armory Kustomize Repo uses kubectl as well but it is highly suggested to use that repo